pipelines/install_pipeline.yml

```diff
@@ -0,0 +1,68 @@
+- hosts: localhost
+  vars_files:
+    - vars/{{ pipeline_type | default('katello') }}_base.yml
```

Comment:
I'm still debating with myself if I want to keep this `default` here or not…

Reply:
Drop it.